pints/toy/_toy_classes.py

```diff
@@ -61,3 +64,155 @@ def suggested_times(self):
         """
         raise NotImplementedError
 
+    def simulate(self, parameters, times):
```

Comment:
This shouldn't be here, as it's already defined in `ForwardModel`

Reply:
(which this class doesn't inherit from)

Reply:
removed this